src/osc_adc.cpp

```diff
@@ -376,7 +376,6 @@ void M2kAdc::setSampleRate(double sr)
 	if(filteringEnabled())
 	{
 		GenericAdc::setSampleRate(sr);
-		setOversamplingRatio(1);
```

Comment:
This line is necessary to prevent the bug fixed in: https://github.com/analogdevicesinc/scopy/commit/2fc96ccec862f21d0706990a31a028e3ca544c42

Reply:
Dropped commit that introduced this